tests/Agent/UnitTests/Core.UnitTest/Segments/MessageBrokerSegmentDataTests.cs

```diff
@@ -18,7 +18,9 @@ public class MessageBrokerSegmentDataTests
     {
         #region IsCombinableWith
 
+#pragma warning disable NUnit1028 // The non-test method is public
```

Comment:
Should we submit a tech debt item so that we don't need this pragma?

Reply:
Another easy refactor. Done.